src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -536,13 +527,10 @@ class FunctionDefinitionGrammar(BaseSegment):
     )
 
 
-class WildcardExpressionSegment(BaseSegment):
+class WildcardExpressionSegment(ansi.WildcardExpressionSegment):
     """An extension of the star expression for Bigquery."""
 
-    type = "wildcard_expression"
-    match_grammar = ansi_dialect.get_segment(
-        "WildcardExpressionSegment"
-    ).match_grammar.copy(
+    match_grammar = ansi.WildcardExpressionSegment.match_grammar.copy(
```

Comment:
This PR eliminates dialects using `get_segment()` in favor of direct attribute access (cleaner).